### PR TITLE
No README.md #2

### LINKED ISSUE #2: No README.md

### PR DESCRIPTION
**Pull request by whughes98144**
Initial cut of README.md.  Needs more expert go over and recommendation on areas to expand on.

May make sense to include/expand some of this in other repo README.md's.

COMMENTED OUT the auth link in build.gradle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,16 +135,21 @@ private var splash: SplashRenderer? = null
     }
 }
 ```
-The most important annotation after @Mixin is the @SourceInjection, which tells you *where* you're putting the code.
-The options for this are:
-
-DURGAN!!!
+The most important annotation after @Mixin is the @SourceInjection (Or field or method injection), which tells you *where* you're putting the code.
```
Comment (durganmcbroom) [inline README.md]: Take out or field/method injection

File: README.md [hunk 2]
```diff
@@ -39,10 +39,8 @@ one of five main strengths of the Yak system.
 Any version partition can call code in the src/main partition but main cannot see anything in the version partitions. 
 Also, again, note that we've restricted any MC-code from the main to insure good design and reduce code-version fragility.
 
-At the simplest, you can develop JUST in a partition directory, nothing in src/main and make an extension that works for 
-only a specific set of MC versions.  
-
-
+At the simplest, you can develop almost everything in the version partition, the only thing necessary in src/main 
+is the Extension classand.
```
Comment (durganmcbroom) [inline README.md]: classand?

File: README.md [hunk 3]
```diff
@@ -113,7 +111,7 @@ using the correct classes for the version you're targeting!
  
 ### Mixins
 In general, mixins are any time code is "mixed into" other code.  And if you've done MC dev you're familiar with these.
-You need to tell *where* you want to put the code (at the beginning of the function?  At some point you searched for
+You need to tell *where* you want to put the code (at the beginning of the function  At some point you searched for
```
Comment (durganmcbroom) [inline README.md]: remove the part after At...

File: README.md (hunk 1 above)
Comment (durganmcbroom) [reply README.md]: Specifically @SourceInjection denotes where the given instructions are going in the target method in the target class. With `point` you can define what part of the method this targets.

Comment (whughes98144) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (whughes98144) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (whughes98144) [reply README.md]: done

**Review by durganmcbroom — APPROVED** [review]

**Pull request merged**